requirements.txt

```diff
@@ -1,2 +1,2 @@
-aiogram
+aiogram>=2.12,<3.0
```

Comment:
wouldn't ~=2.12 mean >=2.12,<3.0 tho?

Reply:
@comictomcat I think ~2.12 doesn't include 2.13, 2.14 and so on... right?

Reply:
the "~=" thingy basically means compatible version 
you can rewrite it as >= V.N, == V.*
meaning that it'll include minor versions such as 2.13, 2,14 ...
but, I guess, it's much cleaner to have something like >=2.12, <3.0